code/modules/antagonists/gang/gang.dm

```diff
@@ -82,10 +82,9 @@
 	..()
 
 /datum/antagonist/gang/greet()
-	to_chat(owner.current, "<B>As you're the first gangster, your uniform and spraycan are in your inventory!</B>")
-
```

Comment:
can you make this one to chat with breaklines instead of many

Reply:
That's way less readable.

Reply:
![image](https://user-images.githubusercontent.com/10467687/85232429-3eb9ee80-b3d5-11ea-85b8-ee1744da59b4.png)

Reply:
alternatively
![image](https://user-images.githubusercontent.com/10467687/85232463-5b562680-b3d5-11ea-94f6-0c7e4354e319.png)

Reply:
and at this point you can just have one bold tag instead of 6

Reply:
bobaah is correct, please do that